src/tritonserver.cc

```diff
@@ -1258,6 +1258,28 @@ TRITONSERVER_ServerOptionsSetLogVerbose(
   return nullptr;  // Success
 }
 
+// Set verbose logging level. Level zero disables verbose logging.
```

Comment:
Fix the comment.

Reply:
Simply removed, as there is detailed description in header